rpm/0002-Implement-Sailfish-OS-specific-privileged-data-optio.patch

```diff
@@ -396,31 +450,22 @@ index fd359c39..5c1676dc 100644
  		if (cfg.chrootdir)
  			fwarning("private-opt feature is disabled in chroot\n");
 diff --git a/src/firejail/usage.c b/src/firejail/usage.c
-index 888a6ffe..098bf696 100644
+index 0a4c8a483..5b376dc3c 100644
 --- a/src/firejail/usage.c
 +++ b/src/firejail/usage.c
-@@ -195,6 +195,7 @@ static char *usage_str =
+@@ -210,6 +210,7 @@ static char *usage_str =
  	"    --private-cwd=directory - set working directory inside jail.\n"
  	"    --private-opt=file,directory - build a new /opt in a temporary filesystem.\n"
  	"    --private-srv=file,directory - build a new /srv in a temporary filesystem.\n"
 +	"    --privileged-data=directory - whitelist privileged user data directory.\n"
  	"    --profile=filename|profile_name - use a custom profile.\n"
  	"    --profile.print=name|pid - print the name of profile file.\n"
- 	"    --profile-path=directory - use this directory to look for profile files.\n"
+ 	"    --protocol=protocol,protocol,protocol - enable protocol filter.\n"
 diff --git a/src/firejail/util.c b/src/firejail/util.c
-index de31ebdd..110a61da 100644
+index a01290cf2..147e5d22d 100644
 --- a/src/firejail/util.c
 +++ b/src/firejail/util.c
-@@ -132,7 +132,7 @@ static void clean_supplementary_groups(gid_t gid) {
- 	int i = 0;
- 	while (allowed[i]) {
- 		gid_t g = get_group_id(allowed[i]);
--	 	if (g) {
-+		if (g != -1) {
```

Comment:
It looks like this is now in `copy_group_ifcont()` and it check for `g` non-zero value but it should check it for not being `INVALID_GID` or `-1`.

Reply:
Changed to != INVALID_GID